ui/app.html

```diff
@@ -5,7 +5,17 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
 
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,400,900" type="text/css">
-  <!-- link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.2/semantic.min.css"></link -->
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script type="text/javascript">window.gaTrackingId=null</script>
+  <script async src="https://www.googletagmanager.com/gtag/js"></script>
+  <script type="text/javascript">
+    if(window.gaTrackingId !== null) {
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {dataLayer.push(arguments);}
```

Comment:
I'm not sure if the `dataLayer` means `window.dataLayer` in line 14.

Reply:
I don't either, this is copy-pasted form google analytic's "how to embed" documentation and I didn't want to edit anything too much. I can add a comment indicating that though